src/System.IO.FileSystem/tests/Directory/Move.cs

```diff
@@ -127,18 +127,6 @@ public void IncludeSubdirectories()
             Assert.True(Directory.Exists(Path.Combine(testDirDest, testDirSubDirectory)));
         }
 
-        [Fact]
-        public void Path_Longer_Than_MaxLongPath_Throws_Exception()
-        {
-            string testDir = GetTestFilePath();
-            Directory.CreateDirectory(testDir);
-            Assert.All((IOInputs.GetPathsLongerThanMaxLongPath(GetTestFilePath())), (path) =>
```

Comment:
This should continue to throw in some way. I would expect either "too long" or "not found" or possibly "invalid" for any of these long paths. It depends on where they get kicked back in the system.